docs/UserGuide.md

```diff
@@ -196,17 +196,35 @@ Format: `edit NAME [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [s/SALARY] [i/INFO]
 
 * Edits the client named `NAME`.
   * `Name` is case-insensitive. E.g. `John` will match `john`.
+  * Spaces in `NAME` will be counted as separate names, unless `NAME` fully matches client's name
+    * Example: `edit John Doe p\88888888` will find clients with names containing `John` and `Doe`,
+                there exists a client with the name `John Doe`
   * The order of the keywords does not matter. e.g. `Hans Bo` will match `Bo Hans`.
-  * Only full words will be matched e.g. `Han` will not match `Hans`.
+* If `NAME` matches multiple clients, you will see a list of clients with matching names
+  * Input the position on the list of the client you wish to edit.
+
+Example:
+Scenario: You have clients named `John Doe` `John Smith` and `John Willams`
+* Running the command `edit n/John d/2020-12-04` will show a list of clients with names containing "John"
+
+![edit_multiple_clients](images/editMultipleClients.png)
+* Typing `1` will edit "John Doe", typing `2` will edit "John Smith" and typing `3` will edit "John Williams"
+* After typing `1`, the previous meeting date of the client with the name `John Doe` to `2020-12-04`
+
+![edit_client_result](images/editClientResult.png)
+
+<div markdown="block" class="alert alert-info">
+
+**:information_source: Notes on `edit`:**<br>
+
 * `DATE` has to be in the format **YYYY-MM-DD**.
 * **At least one** of the optional fields must be provided.
 * Existing values will be updated with the input values.
 * When editing tags, the existing tags of the client will be removed i.e adding of tags is not cumulative.
 * You can remove all the client’s tags by typing `t/` without specifying any tags after it.
 * You can remove the client's info by typing `i/` without specifying any info after it.
 
-Example:
-* `edit n/John Doe d/2020-12-04` Edits the previous meeting date of the client with the name `John Doe` to `2020-12-04` which is 4th Dec 2020.
```

Comment:
Maybe example should be kept to keep the format consistent?

Reply:
Ok added it back in as a separate scenario